src/PackageResolver.php

```diff
@@ -46,7 +46,7 @@ public function resolve(iterable $arguments = []): iterable
         // second pass to resolve package names
         $packages = [];
         foreach ($explodedArguments as $i => $argument) {
-            if (false === strpos($argument, '/')) {
+            if ('php' !== $argument && 0 !== strpos($argument, 'ext-') && false === strpos($argument, '/')) {
```

Comment:
We should probably use [PLATFORM_PACKAGE_REGEX](https://github.com/composer/composer/blob/master/src/Composer/Repository/PlatformRepository.php#L27) to handle all cases and stay in sync to possible future platform packages.